client/app/components/preprint-basics/component.js

```diff
@@ -43,14 +43,35 @@ function doiRegexExec(doi) {
 /* Does not support editing */
 export default Ember.Component.extend(BasicsValidations, {
     editMode: true,
-    uploadValid: Ember.computed.alias('nodeLocked'), // Once the node has been locked (happens in step one of upload section), users are free to navigate through form unrestricted
+    excludeContent: Ember.computed('excludeList', function(){
+        let list = this.get('options.excludeList');
+        return {
+            license: list.includes('license'),
```

Comment:
I get an `Uncaught TypeError: Cannot read property 'includes' of undefined` here when I try to view the Basic Info section of the preprint submission form